mewarpx/mewarpx/diags_store/checkpoint_diagnostic.py

```diff
@@ -0,0 +1,43 @@
+"""Class for installing a checkpoint diagnostic"""
+
+from mewarpx.mwxrun import mwxrun
+from mewarpx.diags_store.diag_base import WarpXDiagnostic
+
+from pywarpx import picmi
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CheckPointDiagnostic(WarpXDiagnostic):
+    def __init__(self, diag_steps, name="chkpoint", write_dir=".", **kwargs):
```

Comment:
```suggestion
    def __init__(self, diag_steps, name="checkpoint", write_dir=".", **kwargs):
```
Small thing but I think that if the abbreviation is just going to be 2 letters shorter than the full word, we might as well use the full word to be more descriptive.

Reply:
ok agreed